### ISSUE TITLE
Installation should stop + clean up if a utility is not available

### ISSUE THREAD

**Issue by aplowman** [hpcflow/install-scripts#56]
E.g. no `unzip`:

![image](https://github.com/hpcflow/install-scripts/assets/5176471/64ff19e1-c59f-4ffe-b8a8-5f11720b1121)

Indicates installation was completed and the app is available, but it is not. If we then install `unzip`, and retry, we are told it is already installed:

![image](https://github.com/hpcflow/install-scripts/assets/5176471/722de28a-8363-4ac7-9c82-dc4f402a3a57)

**Comment by gcapes** [comment]
Perhaps it would be easier to first check whether all the requirements are installed, and abort if something is missing?

**Comment by aplowman** [comment]
Yeah, see also: https://github.com/hpcflow/install-scripts/issues/55
